client/damage/damage.lua

```diff
@@ -165,7 +165,7 @@ local function applyDamage(ped, damageDone, isArmorDamaged)
     if damageDone >= config.healthDamage then
         local isBodyHitOrWeakWeapon = checkBodyHitOrWeakWeapon(isArmorDamaged, bodypart, weaponClass)
         if isBodyHitOrWeakWeapon and isArmorDamaged then
-            lib.callback('qbx_medical:server:setArmor', false, false, GetPedArmour(ped))
+            lib.callback.await('qbx_medical:server:setArmor', false, GetPedArmour(ped))
```

Comment:
We aren't doing anything with the return from this. Should this be a NetEvent?

Reply:
Maybe. It guarantees the function has finished setting the armor before it returns.